scripts/nginx/create_vhost.sh

```diff
@@ -222,6 +222,7 @@ function preparePHPVhostContent() {
     location ~ \.php$ {
         fastcgi_split_path_info ^(.+\.php)(/.+)$;
         fastcgi_pass unix:/var/run/php/php${PHP_VER}-fpm.sock;
+	fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
```

Comment:
you need to escape the '1' sig.. It should be as below:

fastcgi_param SCRIPT_FILENAME **\**$realpath_root**\**$fastcgi_script_name;

Reply:
done